public/src/pages/views/MemberView.tsx

```diff
@@ -78,18 +77,18 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
                 this.state.pendingUsers,
                 [],
                 ActionType.InRow,
-                (user: IUserRelation) => {
-                    return this.generateUserButtons(user.enrollment);
+                (user: Enrollment) => {
```

Comment:
Rename `user` to `enrollment`.

Reply:
Renamed here, renamed in all other cases mentioned below as well.